### PR TITLE
Update pm2 config - Closes #703

### LINKED ISSUE #703: Update PM2 config
### Description

Update the PM2 configuration to keep default settings suitable for the testnet & mainnet networks without issues.

### Acceptance Criteria

- Update default block index to infinite
- Add volatile storage with a default value
- Make sure the indexing starts foo genesis (keep the number of blocks default)

### PR DESCRIPTION
**Pull request by MichalTuleja**
### What was the problem?
This PR resolves #703

### How was it solved?
- Updated ecosystem file

### How was it tested?
Jenkins

### CODE REVIEW

**Review by nagdahimanshu — APPROVED** [review]
Looks good and works well 👍

File: ecosystem.core3.config.js [hunk 1]
```diff
@@ -51,20 +50,19 @@ module.exports = {
...
-				// SERVICE_CORE_REDIS: 'redis://localhost:6379/1',
-				// SERVICE_CORE_MYSQL: 'mysql://root@localhost:3306/sdk_v5_betanet',
+				SERVICE_BROKER: 'redis://localhost:6379/0',
+				LISK_CORE_WS: 'ws://localhost:8080',
+				SERVICE_CORE_REDIS: 'redis://localhost:6379/1',
+				SERVICE_CORE_REDIS_VOLATILE: "redis://localhost:6379/8",
+				SERVICE_CORE_MYSQL: 'mysql://root:password@localhost:3306/lisk',
 				LISK_STATIC: 'https://static-data.lisk.com',
-				GEOIP_JSON: 'https://geoip.lisk.io/json',
+				GEOIP_JSON: 'false',
```
Comment (sameersubudhi) [inline ecosystem.core3.config.js:59]: Why are we setting `GEOIP_JSON` to `false` instead of `''` by default?
```suggestion
				GEOIP_JSON: '',
```

**Review by sameersubudhi — APPROVED** [review]
Everything apart from the following comment looks good and works as expected.

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

**Pull request merged**